screens/PaymentFormScreen.js

```diff
@@ -114,21 +144,45 @@ class PaymentFormScreen extends React.Component {
               </View>
             </View>
 
-            {discounts.map((discount, ix) => (
-              <View
-                style={[styles.borderBottomLine, styles.paddingTopBtn8]}
-                key={ix}
-              >
-                <Field
-                  name="discountPercent"
-                  component={RenderCheckBox}
-                  customValue={discount.value}
-                  optionName={discount.label}
-                  total={order.orderTotal}
-                  getPercent={this.getPercent}
-                />
+            <View style={[styles.paddingTopBtn20, styles.borderBottomLine]}>
```

Comment:
I think the discount Offers section is not needed, just keep the Discount section.